add christensen yuen cookbook (#4228)
geodynamics/aspect: doc/manual/manual.tex

> Finally, in order to just test whether the initial temperature structure has been properly defined, it is helpful to run the model for a single time-step and without actually waiting for the solvers to converge. In order to do this, the \texttt{End time} can be set to zero. If the model is very large (lots of refinement) or there are large viscosity jumps that take longer to converge in the Stokes solver, it can also be useful to set the solver tolerance parameters to large values as noted below. However, remember that in that case the solution will not be converged -- it is only useful for checking that the initial condition is set correctly.

Reviewer: ```suggestion
We can test whether the initial temperature structure was properly defined by running the model for a single time-step, without waiting for the solvers to converge. In order to do this, the \texttt{End time} can be set to zero. If the model is very large (lots of refinement) or there are large viscosity jumps that take longer to converge in the Stokes solver, it can also be useful to set the solver tolerance parameters to large values as noted below. However, remember that in that case the solution will not be converged -- it is only useful for checking that the initial condition is set correctly.
```

Author: That's not related to the changes I am proposing. If you want to improve this other cookbook, you can submit your own pull request :grin: 
We usually try to keep unrelated changes in separate PRs.
